DXMainClient/PreStartup.cs

```diff
@@ -82,6 +82,7 @@ public static void Initialize(StartupParams parameters)
 
             Logger.Initialize(clientUserFilesDirectory.FullName, clientLogFile.Name);
             Logger.WriteLogFile = true;
+            MainClientConstants.LoggerInitialized = true;
```

Comment:
Isn't there a built in way to check if logger is inited? If yes - would prefer that instead, if not - then this should work as is but should be implemented in future as a part of logger class (maybe throw a TODO on the `LoggerInitialized`).